### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.hero-tagline)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14482]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
.hero-tagline
#### Snippet
`<p class="hero-tagline">{Fast, Correct} - Choose two</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.41 (foreground color: #757575, background color: #fafafa, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
